models/balancer/arbitrum/balancer_arbitrum_schema.yml

```
@@ -76,3 +77,39 @@ models:
       - &median_price
         name: median_price
         description: 'BPT median price'
+
+  - name: balancer_arbitrum_pools_tokens_weights
+    meta:
+      blockchain: arbitrum
+      project: balancer
+      contributors: jacektrocinski
+    config:
+      tags: ['arbitrum', 'balancer', 'pools', 'tokens', 'weights']
+    description: >
+      Token weights in Balancer pools.
+    columns:
+      - &pool_id
+        name: pool_id
+        description: 'Unique encoded identifier that refers to each pool'
+        tests:
+          - not_null
+      - &token_address
+        name: token_address
+        description: 'Contract address for the token'
+      - &normalized_weight
+        name: normalized_weight
+        description: 'Weight of the token in the pool.'
+
+  - name: balancer_v2_arbitrum_pools_tokens_weights
+    meta:
+      blockchain: arbitrum
+      project: balancer
+      contributors: jacektrocinski
+    config:
+      tags: ['arbitrum', 'balancer', 'pools', 'tokens', 'weights']
+    description: >
+      Token weights in Balancer v2 pools.
+    columns:
+      - *pool_id
```

You can add an uniqueness test here.
```suggestion
      project: balancer
      contributors: jacektrocinski
    config:
      tags: ['arbitrum', 'balancer', 'pools', 'tokens', 'weights']
    description: >
      Token weights in Balancer v2 pools.
    tests:
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - pool_id
            - token_address
    columns:
      - *pool_id
```